src/frontend/src/lib/components/hero/Actions.svelte

```diff
@@ -30,6 +30,9 @@
 
 	let convertBtc = false;
 	$: convertBtc = $tokenCkBtcLedger && $erc20UserTokensInitialized;
+
+	let isTokenPage = false;
+	$: isTokenPage = isRouteTransactions($page);
```

Comment:
Maybe a naming to review? 

isToken... vs is...Transaction

Or `isRouteTransactions` is actually incorrect?

Reply:
right! changed to `isTransactionsPage` (even if ultimately probably it will become a token page that page... but it's the story for another time)